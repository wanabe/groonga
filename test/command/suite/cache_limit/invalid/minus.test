cache_limit --max -1
