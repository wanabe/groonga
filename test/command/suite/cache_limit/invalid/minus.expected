cache_limit --max -1
[[[-22,0.0,0.0],"max value is invalid unsigned integer format: <-1>"],100]
#|e| max value is invalid unsigned integer format: <-1>
